app/views.py

```diff
@@ -183,8 +180,8 @@ def login():
 @app.route('/logout')
 def revoke_token():
     if 'google_token' in session:
-        res = google.get('https://accounts.google.com/o/oauth2/revoke',
-                         data={'token': session['google_token'][0]})
+        google.get('https://accounts.google.com/o/oauth2/revoke',
```

Comment:
It's also extremely useful to get in the habit of avoiding assignment to any variables that you don't end up using. Here `res` is never referred to again, so there's no need to bind the result of `google.get()`. Either we are calling to `get` for its side effects, *or* this is actually an unnecessary call that can be removed.